products/rhel8/profiles/stig.profile

```diff
@@ -647,6 +647,7 @@ selections:
     - no_empty_passwords
 
     # RHEL-08-020332
+    - accounts_passwords_pam_disable_nullok
```

Comment:
You should also update the profile stability tests in `tests/data/profile_stability/rhel8/`, for both stig and stig_gui profiles.